src/core/components/Search/SearchResult.js

```diff
@@ -0,0 +1,28 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+
+
+export default class SearchResult extends React.Component {
+  static propTypes = {
+    result: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { result } = this.props;
+    return (
+      <li className="SearchResult">
+        <Link to={`/en-GB/firefox/addon/${result.slug}/`}
```

Comment:
Locale again, links on this app are going to get unwieldy fast if we don't make this easier.